new_buildpack.go

```diff
@@ -0,0 +1,104 @@
+package pack
+
+import (
+	"context"
+	"os"
+	"path/filepath"
+	"runtime"
+
+	"github.com/BurntSushi/toml"
+
+	"github.com/buildpacks/pack/internal/build"
+	"github.com/buildpacks/pack/internal/dist"
+	"github.com/buildpacks/pack/internal/style"
+)
+
+var (
+	bashBinBuild = `
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+layers_dir="$1"
+env_dir="$2/env"
+plan_path="$3"
+
+exit 0
+`
+	bashBinDetect = `
+#!/usr/bin/env bash
+
```

Comment:
Is there any benefit to setting `-euo pipefail` in the detect script as well?

Reply:
I went back and for. Since it's only `exit 0`, no. I'd rather keep it lean.